sigs.yaml

```
@@ -2602,7 +2602,7 @@ workinggroups:
     time: "13:00"
     tz: PT (Pacific Time)
     frequency: biweekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: no-zoom-link
```

```suggestion
    url: https://zoom.com.cn/j/103404077
```

From agenda: https://docs.google.com/document/d/1t2zTAehIEu3Xy8VJ6dc_cgODO4WGN-FDwpAfBjYCKc4/edit